docs/adr/0008-redis-lua-scripts.md

```diff
@@ -0,0 +1,75 @@
+# Redis Lua Scripts
+
+- Deciders: dannycoates, et. al
```

Comment:
This becomes somewhat circular, but anyone reviewing this should probably be added as a decider.